lib/test/s-activity-indicator.visual.test.ts

```diff
@@ -0,0 +1,35 @@
+import { html } from "@open-wc/testing";
+import { getTestVariations, makeTest, makeTestElement } from "../ts/test-utils";
+import "../ts/index";
+
+describe("s-activity-indicator", () => {
+    getTestVariations({
```

Comment:
I wonder if we can abstract even further and hide some more implementation details.
Some pseudo code
```
import { generateVisualTests, generateA11yTests } from 'test-utils'

generateVisualTests({
  ...variationsParameters, // baseClass, variants, etc...
  extraVariations: string[] // to capture things like ["", "new"]
  template: ({ classes,  with a fn we could pass here pretty much anything including a prefilled version of the makeTestElement fn if necessary }) => html`... ${class}...`
})
```

I hope this makes sense. The base functions you have created are great and should be reused. I am suggesting to create an "abstraction of the abstraction" to make things even simpler for the consumers.

Reply:
I added `runComponentTests` (note the plural) to abstract out some common patterns. I _could_ add `runA11yComponentTests` and `runVisualComponentTests` but that doesn't seem too much more convenient then specifying a `type` argument. Curious to get your thoughts on the abstractions!

Sidenote: Naming things is tough so I'd like to take a moment when this is close to being merged to reconsider the util function naming.